travis/publish.sh

```diff
@@ -104,6 +104,50 @@ safe_checkout_master() {
   fi
 }
 
+test_server() {
+  # Test Stackdriver Storage module with the latest version of Zipkin server.
+  temp_dir=$(mktemp -d)
+  pushd $temp_dir
+
+  # Download and unpack Zipkin Server
+  wget https://jitpack.io/com/github/openzipkin/zipkin/zipkin-server/master-SNAPSHOT/zipkin-server-master-SNAPSHOT-exec.jar
+
+  # Zipkin server JAR is a self-executable with headers that causes unzip to exit with status 1
+  # Fix the archive first.
+  zip -A zipkin-server-master-SNAPSHOT-exec.jar
+  unzip zipkin-server-master-SNAPSHOT-exec.jar -d zipkin
+
+  # Unpack our module
+  unzip -n $TRAVIS_BUILD_DIR/autoconfigure/storage-stackdriver/target/*-module.jar -d zipkin
```

Comment:
I think you can set both jars to `-cp` instead of unzipping

Reply:
that didn't work for me. I was wondering why unzip was necessary in Zipkin's Dockerfiles, but apparently, can't do it w/ simple `java -cp`

Reply:
unzip is used in zipkin's dockerfiles as it speeds up bootstrapping and allows layering to play games with the contents.

Reply:
I'm surprised what didn't work, but we can try to pin down what it was. I think in one scenario we had to unzip due to a nested compressed jar problem? @jeqo do you remember?

https://github.com/openzipkin-contrib/zipkin-storage-kafka/pull/30

Reply:
Ah yeah the spring loader.path property doesn't support nested jars which is annoying. But that's mainly a problem for the zipkin-server docker image which supports arbitrarily named jars as plugins. Since we know the names here I thought `-cp` should work without issues (I don't unzip when doing this sort of command locally) but not a big deal to unzip anyways.

Reply:
We have nested jars in other repos. -cp isn't the integration path for this, it is what is in the readme (loader path) which is capable of nested jars. IIRC there was a problem with "compressed" jars, but that wasn't entirely validated either.

I routinely run the README for example, which had been working, so either we're not noticing something subtle here or there's an env thing..

Reply:
i updated the command line to use the jar. it was my bad i didn't fully understand how to refer to all the jars differently. ptal the new command line.

Reply:
changed to match the README